some small fixes (documentation/code) (#2205)
smallrye/smallrye-graphql: docs/typesafe-client-usage.md

## Namespaces

There are several ways to work with namespaces in a type-safe client:
1. Using `@Namespace`
2. Using `@Name` (deprecated)

> > [NOTE] You can only use one of the annotations - `@Name` or `@Namespace` over the GraphQLApi classes.

Reviewer: ```suggestion
> [NOTE] You can only use one of the annotations - `@Name` or `@Namespace` on a `GraphClientQLApi` interface.
```
